src/params.rs

```diff
@@ -52,7 +52,7 @@ impl Params {
             Some(filetypes) => vec![
                 self.get_directory().unwrap(),
                 String::from("**"),
-                format!("{{{}}}", filetypes),
+                format!("*.{{{filetypes}}}"),
```

Comment:
This is the format that should be used with globwalk, see [documentation](https://docs.rs/globwalk/0.8.1/globwalk/index.html)